i/invariant/invariant_ubi_8.5.sh

```diff
@@ -0,0 +1,51 @@
+# -----------------------------------------------------------------------------
+# Package		: invariant
+# Version		: 2.2.4
+# url       	: https://github.com/zertosh/invariant
+# Tested on		: "Red Hat Enterprise Linux 8.5" (Docker)
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Saurabh Gore <Saurabh.Gore@ibm.com> / <smohite@us.ibm.com> 
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+WORK_DIR=`pwd`
+
+PACKAGE_NAME=invariant
+PACKAGE_VERSION=v2.2.4                 
+PACKAGE_URL=https://github.com/zertosh/invariant.git
+
+
+dnf install git wget -y
+
+# install nodejs
+dnf module install nodejs:10 -y
```

Comment:
Can you please try to build validate this using nodejs 12 ?

Reply:
@kandarpamalipeddi  Updated script with nodejs 12, Please review.